Deploy/terraform/environments/ab2d-mgmt-east-dev/authorized_keys

```diff
@@ -0,0 +1,11 @@
+# Keys included with gold image
+
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDVbSnh++tT/inkRLwrVDO3rx7fBctAogNMLRYU8A9DKpncAa69gi9Nvb+JyrMpk9TjnAQo/8k2iGIx2d9tiHrrMi3LtP3jeMTW4Nyr5gtW3NxzmChd01R0KDkRqJj8GOpklX9X9fOQADc4dVKO3Tat6MIfsUAlkVPcv3UtIHPQ0BZ3h81++onb8ImTb41E/ehUSx5STFxXL3LXwnZnEn5UkqaeQfwOtZfHUChTvKviHMKJb5VJl1RmoXjOiotukLJamHsUpEr5eloT8Dy9MKnx1NEOeC2xenzSnRd3Kl8IJsxW6B8+dF8pstSDhQm/utvi4n17pY1Ldq+eJzt5WWCt
```

Comment:
Is this key a security issue?

Reply:
Will be excluded in a later pull request.